tests/integration/components/task/archive-task-test.js

```diff
@@ -26,6 +27,9 @@ moduleForComponent('task/archive-task', 'Integration | Component | task/archive
 });
 
 test('can archive if user has ability', function(assert) {
+  let { confirm } = window;
+  window.confirm = () => true;
```

Comment:
lmk what you think of this pattern.  I can try using sinon as well!

Reply:
@snewcomer We've actually worked on removing this one in favor of using sinon. Just seems safer than this.

We have an example in `tests/acceptance/admin-organization-invite-new-test.js`

Reply:
@begedin ah thanks!  Fixed.